0023-bilateral-transfer-protocol/0023-bilateral-transfer-protocol.md

```diff
@@ -232,7 +209,7 @@ For example, the primary sub-protocol entry of a Message might represent a quote
 
 Likewise, only the primary sub-protocol data in a Response indicates whether result of the request from the Message being responded to actually succeeded or not.
 
-In Error, Prepare, Fulfill, and Reject calls, the distinction between primary and secondary sub-protocol entries is less strict.
+In Error calls, the distinction between primary and secondary sub-protocol entries is less strict.
 
 ## Flow
 
```

Comment:
"The request types are `Message`, `Prepare`, `Fulfill` and `Reject`" should be "The
 request types are `Message` and `Transfer`"